test/integration/targets/ecs_certificate/aliases

```diff
@@ -0,0 +1,15 @@
+# Not enabled due to lack of access to test environments. May be enabled using custom integration_config.yml
+# Example integation_config.yml
+# ---
+# entrust_api_user: apiuser-99543
```

Comment:
I hope that these aren't the real values. Asking just in case :)

Reply:
Good point. Replacing the data by more obvious data (`apiuser-1234`, `hunter2`) should also be ok.